tools/challenge-auditor/index.ts

```diff
@@ -0,0 +1,90 @@
+import { access, readdir } from 'fs/promises';
+import { join } from 'path';
+
+import { availableLangs, auditedCerts } from '../../config/i18n/all-langs';
+
+const superBlockFolderMap = {
+  'responsive-web-design': '01-responsive-web-design',
+  'javascript-algorithms-and-data-structures':
+    '02-javascript-algorithms-and-data-structures',
+  'front-end-development-libraries': '03-front-end-development-libraries',
+  'data-visualization': '04-data-visualization',
+  'back-end-development-and-apis': '05-back-end-development-and-apis',
+  'quality-assurance': '06-quality-assurance',
+  'scientific-computing-with-python': '07-scientific-computing-with-python',
+  'data-analysis-with-python': '08-data-analysis-with-python',
+  'information-security': '09-information-security',
+  'machine-learning-with-python': '10-machine-learning-with-python',
+  'coding-interview-prep': '11-coding-interview-prep',
+  'relational-database': '13-relational-database',
+  '2022/responsive-web-design': '14-responsive-web-design-22',
+  '2022/javascript-algorithms-and-data-structures':
+    '15-javascript-algorithms-and-data-structures-22'
+};
+
+const blocksThatNeedToMove = ['d3-dashboard'];
```

Comment:
```suggestion
// These blocks are in the incorrect superblock. They should be moved but, for
// the audit, we just ignore them.
const blocksThatNeedToMove = ['d3-dashboard'];
```
It took me a while to figure out what this was for, so I think a comment could help.